### DISCUSSION TITLE
Android 13 Jelly Star xDrip-plus-20240419-645e8b4

### DISCUSSION THREAD

**Discussion by edneville** [NightscoutFoundation/xDrip#3599] [Bug reports]
Hi,

This can install on my moto g13 (Android 13), but on a Jelly Star (Android 13, build Jelly_Star_EEA_20230627) I get:

"Unsafe app blocked

This app was built for an older version of Android and doesn't include the latest privacy protections."

I select install anyway, but it then says 

"The app wasn't installed".

Any ideas?

**Comment by Navid200** [comment]
Are you sure you don't have an unofficial xDrip already installed?

**Comment by edneville** [comment]
Hi @Navid200, thanks for responding. I've not seen anything named 'xdrip' in the installed applications. We've put librelink and gluroo on, mysugr and glooko on. These shouldn't conflict should they?

**Reply by Navid200** [reply]
No, those should not interfere.

Can you disable play protect and see if it works then?
https://support.google.com/googleplay/answer/2812853?hl=en

**Comment by edneville** [comment]
On 2024-07-28 10:39-0700, Navid wrote:
> No, those should not interfere.
> 
> Can you disable play protect and see if it works then?
> https://support.google.com/googleplay/answer/2812853?hl=en

That's it! Thanks - not sure why the two phones had different account 
settings. Much appreciated!

**Comment by edneville** [comment]
Thanks @Navid200, your solution solved this.

**Reply by Navid200** [reply]
You're welcome
Thanks for the feedback
